File: src/me/Dinner1111/School/Utils/ArrayUtils.java
package me.Dinner1111.School.Utils;

public class ArrayUtils {
	/**
	 * Checks if there are duplicate objects in the same array
	 * @param array The object array
	 * @return If there are duplicates
	 */
	public static final boolean checkDuplicates(Object[] array) {
		Object[] arrayCopy = array.clone();
		for (int arrayIndex = 0; arrayIndex < array.length; arrayIndex++)
			for (int copyIndex = 0; copyIndex < arrayCopy.length; copyIndex++)
				if (!array[arrayIndex].toString().equals(arrayCopy[copyIndex].toString())) continue;
				else return false;
		return true;
	}
}

File: School/Utils/ArrayUtils.cs
using System;

namespace School.Utils {
    class ArrayUtils {
        /**
         * Checks if there are duplicate objects in the same array
         * @param array The object array
         * @return If there are duplicates
         */
        public static bool checkDuplicates(Object[] array) {
            Object[] arrayCopy = (Object[]) array.Clone();
            for (int arrayIndex = 0; arrayIndex < array.Length; arrayIndex++)
                for (int copyIndex = 0; copyIndex < arrayCopy.Length; copyIndex++)
                    if (!array[arrayIndex].ToString().Equals(arrayCopy[copyIndex].ToString())) continue;
                    else return false;
            return true;
        }
    }
}
